<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmpDialog</class>
 <widget class="QDialog" name="AddEmpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter   un emprunteur</string>
  </property>
  <widget class="QTextEdit" name="fNameText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : pink</string>
   </property>
   <property name="text">
    <string>    Pr√©nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : pink</string>
   </property>
   <property name="text">
    <string>   Nom</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lNameText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : pink</string>
   </property>
   <property name="text">
    <string>    Date de naissance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : pink</string>
   </property>
   <property name="text">
    <string>    Adresse</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="adressText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : pink</string>
   </property>
   <property name="text">
    <string>    Travail</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="professionText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Utilisez ce formulaire pour ajouter un emprunteur.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anuuler">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
